dynamic_perks/hooks/scenarios/world/starting_scenario.nut

```diff
@@ -0,0 +1,5 @@
+::DynamicPerks.HooksMod.hook("scripts/scenarios/world/starting_scenario", function (q) {
+	q.onBuildPerkTree <- function( _bro )
```

Comment:
Perhaps it would be better to pass `_perkTree` as an argument instead of `_bro`. Because then we can always trigger this event whenever any perk tree is generated during that scenario, irrespective of whether that perk tree was attached to a bro or not. @Darxo Thoughts?

Reply:
I decided to go with `_perkTree`.